src/html/m_dflist.cpp

```diff
@@ -35,16 +35,17 @@ TAG_HANDLER_BEGIN(DEFLIST, "DL,DT,DD" )
 
         if (tag.GetName() == wxT("DL"))
         {
-            if (m_WParser->GetContainer()->GetFirstChild() != nullptr)
+            if (m_WParser->GetContainer()->GetFirstChild() != nullptr || !m_WParser->GetContainer()->GetId().IsEmpty())
```

Comment:
Similarly, it would probably make sense to add `HasId()` function and use it here as
```suggestion
            if (m_WParser->GetContainer()->GetFirstChild() != nullptr || m_WParser->GetContainer()->HasId())
```

Reply:
I added `HasId`.